GameData/002_CommunityPartsTitles/Localization/CPT_Mk2Expansion_loc.cfg

```diff
@@ -51,55 +50,52 @@ Localization
 
 		// J-58 "Whiplash"
 		#LOC_M2X_Ramjet_title  = J-59 "Afterburn" TurboRamjet Engine               // J-58 "Afterburn" TurboRamjet Engine
-		#LOC_M2X_mule_title    = JE-1 'Mule' Turbofan Engine                       // JE-1 'Mule' Turbofan Engine
+		#LOC_M2X_mule_title    = J-E1 'Mule' Turbofan Engine                       // JE-1 'Mule' Turbofan Engine
 
 		// JR-7 R.A.P.I.E.R. Engine
-		#LOC_M2X_ESTOC_title   = JR-9 E.S.T.O.C. Engine                            // Z0-OM E.S.T.O.C. Engine
-		#LOC_M2X_MATTOCK_title = JR-9X M.A.T.T.O.C.K. Engine                       // X-44 M.A.T.T.O.C.K. Engine
+		#LOC_M2X_ESTOC_title   = JR-8 E.S.T.O.C. Engine                            // Z0-OM E.S.T.O.C. Engine
```

Comment:
why 9 -> 8 ?

Reply:
Because Mk3Expansion adds engines JR-##, bigger than this, and I don't want to use JR-10...